instructor/common.py

```diff
@@ -98,6 +98,11 @@ def get_pages(assignment):
         if page == assignment.last_page_name:
             enabled = False
 
+    if not assignment.has_wb:
+        wb_pages = page_order[-4:]
```

Comment:
This works, but I'm curious why you didn't factor the wb pages our into their own list, like micro_pages and facs_pages